examples/data-objects/client-ui-lib/package.json

```diff
@@ -51,7 +51,7 @@
     "@fluid-example/fluid-object-interfaces": "^0.27.0",
     "@fluid-example/search-menu": "^0.27.0",
     "@fluid-internal/client-api": "^0.27.0",
-    "@fluidframework/common-utils": "^0.23.0",
+    "@fluidframework/common-utils": "^0.24.0-0",
```

Comment:
would be good to do the bump in a seperate review

Reply:
So lazy class is not exported in previous version of common-utils, so need to bump. Or are you saying bump first and then make modifications to localchannelcontext?

Reply:
Done